spec/features/user_past_order_display_spec.rb

```diff
@@ -1,31 +1,30 @@
 require "rails_helper"
 
-feature "a user's past orders" do 
+feature "an authenticated user's past orders" do 
   before(:each) do 
-  category = Category.create(name: "Stout", description: "warm, rich, chocolate undertones")
-  beer1 = Beer.create(name: "Yeti", state: true, category_id: category.id, description: "big, roasty malt flavor with rich caramel and toffee notes", price: 606, created_at: Date.parse("2012-03-25 09:54:09 UTC"), updated_at: Date.parse("2012-03-25 09:54:09 UTC"))
-  beer2 = Beer.create(name: "Chocolate Stout", state: true, category_id: category.id, description: "full-bodied flavor with hints of rich oatmeal and dark chocolate", price: 612, created_at: Date.parse("2012-03-25 09:54:09 UTC"), updated_at: Date.parse("2012-03-25 09:54:09 UTC"))
-  user = User.create({fullname: "Wolfgang Mozart", username: "Wolfie", email: "MCProdigy@gmail.com", password: "password", password_confirmation: "password", phone_number: "303-675-1234", role: 0})
-  order = Order.create(user_id: user.id, status: "completed", total: 1218, created_at: Date.parse("2012-03-25 09:54:09 UTC"), updated_at: Date.parse("2012-03-25 09:58:15 UTC") )
-  visit root_path
-  click_link "Login"
-  fill_in "user[email]", with: "MCProdigy@gmail.com"
-  fill_in "user[password]", with: "password"
-  click_button "Login"
+    beer1 = {name: "Yeti", state: true, description: "big, roasty malt flavor with rich caramel and toffee notes", price: 606, created_at: Date.parse("2012-03-25 09:54:09 UTC"), updated_at: Date.parse("2012-03-25 09:54:09 UTC")}
+    beer2 = {name: "Chocolate Stout", state: true, description: "full-bodied flavor with hints of rich oatmeal and dark chocolate", price: 612, created_at: Date.parse("2012-03-25 09:54:09 UTC"), updated_at: Date.parse("2012-03-25 09:54:09 UTC")}
+    user = User.create({fullname: "Wolfgang Mozart", display_name: "Wolfie", email: "MCProdigy@gmail.com", password: "password", password_confirmation: "password", phone: "303-675-1234", role: 0})
```

Comment:
Line is too long. [196/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.